Cargo.toml

```diff
@@ -200,6 +201,8 @@ cid = { version = "0.10.1", default-features = false, features = [
 
 frc42_dispatch = "6.0.0"
 
+strum_macros = "0.25.2"
```

Comment:
You already have `strum` in the workspace, which exports the macros; no need to import this crate specifically.